### ISSUE TITLE
iOS 9 warning

### ISSUE THREAD

**Issue by Kaspik** [jaydee3/JDStatusBarNotification#53]
iOS 9 needs:

```
#if __IPHONE_OS_VERSION_MAX_ALLOWED < 90000
    - (NSUInteger)supportedInterfaceOrientations {
#else
    - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
#endif
```

instead of

```
 - (NSUInteger)supportedInterfaceOrientations {
```

please correct this warning, thanks!

**Comment by marioradonic** [comment]
Could you please create cocoapod update for this? Thanks

**Comment by jaydee3** [comment]
PR please?

**Comment by Kaspik** [comment]
@jaydee3 https://github.com/jaydee3/JDStatusBarNotification/pull/57

**Comment by Kaspik** [comment]
@jaydee3 When do you plan update your CocoaPods version?

**Comment by jaydee3** [comment]
can you send another PR for that? I really don't have much time for that atm.

**Comment by Kaspik** [comment]
@jaydee3 What should I do for this?

**Comment by jaydee3** [comment]
ok I just published 1.5.3
